tutorials/detecting-changes-in-sentinel-1-imagery-pt-4/Tutorial_pt_4.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<ul><li>"GEE Sentinel-1 Archive": can you be specific by hyperlinking to the catalog page -- is it (?):</li></ul><p><code>[GEE Sentinel-1 Archive](https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S1_GRD)</code></p><ul><li>"exploration,  the": remove extra space </li><li>"first three parts": it might be helpful to new readers to have hyperlinks, suggest:</li></ul><p><code>([Part 1](https://developers.google.com/earth-engine/tutorials/community/detecting-changes-in-sentinel-1-imagery-pt-1), [Part 2](https://developers.google.com/earth-engine/tutorials/community/detecting-changes-in-sentinel-1-imagery-pt-2), [Part 3](https://developers.google.com/earth-engine/tutorials/community/detecting-changes-in-sentinel-1-imagery-pt-3))</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>